magma/magma
# chore(nms): Upgrade sequelize and umzug

## Summary

Upgrades sequelize and umzug to the latest version.

- fixes #13123 

## Test Plan
* Create and edit organisations 
    * make sure you can navigate to the newly created organisations
* Destroy the NMS docker volume and set it up from scratch make sure the migrations are executed
    * Run `dev_setup.sh` 
    * Make sure `magma-test` exists and can be accessed  
    * Restart the NMS docker container with destroying the volume, make sure the migration are not executed
